src/uucore/src/lib/features/checksum.rs

```diff
@@ -321,10 +322,10 @@ fn determine_regex(lines: &[String]) -> Option<(Regex, bool)> {
 
 // Converts bytes to a hexadecimal string
 fn bytes_to_hex(bytes: &[u8]) -> String {
-    bytes
-        .iter()
-        .map(|byte| format!("{byte:02x}"))
-        .collect::<String>()
+    bytes.iter().fold(String::new(), |mut bytes, byte| {
```

Comment:
Misleading variable name.

```suggestion
    bytes.iter().fold(String::new(), |mut hex, byte| {
```